# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/vr/service/VRService.cpp
# Commit: 75b04de7e99c
# Full Hash: 75b04de7e99cab7bed5cc8b0f2f6d52ddf174bf3
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-05-25 09:39:13
# Regressor Bug: 1550422
# File Overlap Count: 1
# Description:
#   Bug 1550422 - P15. Move Skip and Once gfxPrefs to StaticPrefs. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31259
# ==============================================================================

diff -r 91c3acdb2454 -r 75b04de7e99c gfx/vr/service/VRService.cpp
--- a/gfx/vr/service/VRService.cpp	Fri May 24 11:32:20 2019 +0000
+++ b/gfx/vr/service/VRService.cpp	Fri May 24 11:32:54 2019 +0000
@@ -52,7 +52,7 @@
 already_AddRefed<VRService> VRService::Create() {
   MOZ_ASSERT(NS_IsMainThread());
 
-  if (!gfxPrefs::VRServiceEnabled()) {
+  if (!StaticPrefs::VRServiceEnabled()) {
     return nullptr;
   }
 
@@ -73,7 +73,7 @@
 #if defined(XP_WIN)
       mMutex(NULL),
 #endif
-      mVRProcessEnabled(gfxPrefs::VRProcessEnabled()) {
+      mVRProcessEnabled(StaticPrefs::VRProcessEnabled()) {
   // When we have the VR process, we map the memory
   // of mAPIShmem from GPU process.
   // If we don't have the VR process, we will instantiate